db/schema.sql

```diff
@@ -449,6 +449,7 @@ CREATE TABLE `api_key` (
   `created` datetime NOT NULL,
   `disabled` datetime default NULL,
   `reason` text NOT NULL,
+  `estimated_usage` mediumint(9) NOT NULL,
```

Comment:
The schema change doesn't match the migration; if the migration is correct, this should also be `int(9)` and have a `DEFAULT 0`.